src/ReadVTK.jl

```diff
@@ -433,6 +439,29 @@ function get_data_section(vtk_file::VTKFile, section)
   return VTKData(names, data_arrays, vtk_file)
 end
 
+"""
+    get_field_data(vtk_file::VTKFile)
+
+Retrieve a lightweight `VTKData` object with the field data of the given VTK file.
+
+See also: [`VTKData`](@ref), [`get_point_data`](@ref), [`get_cell_data`](@ref)
+"""
+function get_field_data(vtk_file::VTKFile)
+  names = String[]
+  data_arrays = XMLElement[]
+
+  # Iterate over XML elemens in the section
+  for xml_element in child_elements(field_data(vtk_file))
+    # We do not know how to handle anything other than `DataArray`s
+    @assert LightXML.name(xml_element) == "DataArray"
```

Comment:
```suggestion
    LightXML.name(xml_element) != "DataArray" && continue
```

See https://github.com/JuliaVTK/ReadVTK.jl/pull/65